### ISSUE TITLE
Timer in the online UI

### ISSUE THREAD

**Issue by EYALSHTARK** [Geektoolkit/Dynaframe3#123]
Can you add in the UI a timer which lets me decide the hours in a day in which Dynaframe will work?
I would like it to go to sleep during the night.

Thank you.

**Comment by EYALSHTARK** [comment]
I just saw that its already on disscission. 
Will wait patiently for this feature.

**Comment by Geektoolkit** [comment]
Yes it's on the list, high up, but not in yet. It'll happen after the database feature is finished.

**Comment by Geektoolkit** [comment]
This isn't going to make dynaframe 2 as I'm working on it. I'm planning it for Dynaframe pro, my next version which requires a rewrite. I'll keep this open so the community can consider coding it up.
